feat: Update content about seat deduplication mechanism PLUTO-678 (#1829)
codacy/docs: docs/account/emails.md

To update the email addresses associated with your Codacy account, do the following:

> 1.  Make sure you [set your Git email address](#git-config) so that commits are attributed to you.

Author: Yes, it makes sense to use the imperative in a procedure step. 👍 
I was using "Make sure" because the step is not part of the initial procedure, it can be seen more as a recommendation (I even considered moving to an admonition instead). However, I opted to include this step in the procedure for updating the email addresses, so it should be a direct instruction.

Also, I agree on keeping the contextual information on a separate line, but I'll keep it on the same line due to its length (very short).

> 1.  Make sure you [set your Git email address](#git-config) so that commits are attributed to you.

Author: ```suggestion
1.  [Configure your Git email address](#git-config). This ensures that commits are attributed to you.
```
